app/components/public/speaker-item.js

```diff
@@ -1,7 +1,26 @@
 import Ember from 'ember';
 
-const { Component } = Ember;
+const { Component, computed } = Ember;
 
 export default Component.extend({
-  classNames: ['column', 'four', 'wide']
+  classNameBindings: ['style'],
+
+  style: computed(function() {
+    const isFeatured = this.get('isFeatured');
+    if (isFeatured) {
+      return 'column eight wide';
+    } else {
+      return 'column four wide';
+    }
+  }),
+  didInsertElement() {
+    const isFeatured = this.get('isFeatured');
+    if (isFeatured) {
+      this.$('.speaker-item').addClass(' ui grid stackable');
```

Comment:
if we introduce the classes in the template it will be affecting the `**Speakers**` section too, these classes should only be added if the section is for featured speakers. That is why I have handled it in JS.
Please check this 
![screenshot from 2017-05-10 13-13-42](https://cloud.githubusercontent.com/assets/12807846/25888131/8aea54c2-3582-11e7-8ea2-472eb66d7ec3.png)